### DISCUSSION TITLE
Success with 2021 square pump Miami

### DISCUSSION THREAD

**Discussion by bubble07** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#221] [General]
Hi Everyone,
Just a quick note to say thanks to @visualapproach and all the other contributors. I've got the spa remote working on my 2021 Miami.

3 things to note, hopefully for the benefit of others:
1. 2021 square pump Miami is pump model S100102.
2. I needed the resistors between the board and the screen.
3. When you turn it on and the lights flash randomly for about 10 seconds, it's nothing to panic about and turn it off again, it's just the ESP booting, or at least that's what I think it is...

Thanks again all, it's been a great experience as my first project using GitHub and also first using ESP modules.👍

**Comment by visualapproach** [comment]
Thank you @bubble07 ! I updated the table in the instructions. Good work on your first project 👍

**Comment by markbellkosel** [comment]
Same model here too! Also first project! I took was freaked about the random lights. 

@bubble07 are you referring to the 10k resistor? I have mine up and running without it. perhaps I do need it because it seems to be a little bit flaky.

**Reply by bubble07** [reply]
Hi.
I did include the 10k resistor just in case, but I was meaning the 560 Ohm resistors between the voltage converters and the screen.

Or at least I think I did...now I think about it, it's possible that it was just doing it's boot sequence of lights and I jumped to the conclusion that I needed the resistors. either way, it wasn't too much hassle to put them in and it works now 🙂

**Reply by BIIINO** [reply]
Hi,

where exactly did u attach the 10K resistor? Please do u have any picture of it ? I dont know how to use it. Thank u in advance

**Reply by bubble07** [reply]
Hi Biiino, sorry for the slow reply. I put the 10k in here (this image from the documentation doesn't have it in, but it's marked on the PCB here(.
![pcb~2](https://user-images.githubusercontent.com/99989342/178101553-f71e675f-b99c-46bc-b77c-ea00ca1d093b.jpg)

**Reply by urbanframe** [reply]
Hi all,

I'm helping a friend build the module for a S100102 pump. 

Please can you confirm which level converter you are having success with?

I'm currently testing with a blue TXS0108E and having issues. 

Also, did you have to wire as per page 7 of the pdf guide?

![image](https://user-images.githubusercontent.com/8011551/227791289-3c696626-7521-4dcc-bb77-f1d6f2f81007.jpeg)


Thanks for your help.

**Reply by chumleychuff** [reply]
Did you ever build this? I have the parts arriving and the same model pump
